### ISSUE TITLE
No results shown for many queries - Too large text field error

### ISSUE THREAD

**Issue by joepio** [openstate/open-raadsinformatie#444]
I heard that Utrecht didn't show any docs. Turns out that there's an error that keeps popping up:

```
"reason": {
                            "type": "illegal_argument_exception",
                            "reason": "The length of [text] field of [785816] doc of [ori_utrecht_20230707003938] index has exceeded [1000000] - maximum allowed to be analyzed for highlighting. This maximum can be set by changing the [index.highlight.max_analyzed_offset] index level setting. For large texts, indexing with offsets or term vectors is recommended!"
                        }
```

**Comment by breyten** [comment]
It seems that this only happens when sorting on relevance:
![image](https://github.com/openstate/open-raadsinformatie/assets/154791/f9b0af7a-4734-4d5d-885d-74797815d894)

When sorting on newest you get the following:
![image](https://github.com/openstate/open-raadsinformatie/assets/154791/bbcdaf18-87fe-418c-9d94-5c44b9152d7a)

In general this confuses me. This has not changed since it went offline, so it has always been there.

Indexing with term vectors or offsets would probably require a new loading of the data :/

**Comment by joepio** [comment]
Hmmm. IIRC we simply ignored documents that were too large before (mostly "agendabundels" which were duplicates anyway). Perhaps this has changed?

**Comment by joepio** [comment]
Bug is still there, even with default sort oder set to date. See #457.

I think this needs a different type of fix during indexing.

**Comment by breyten** [comment]
Fixed with 57e5de9cf18e69a2137e9a713eadf609ab0c6c7c
